packages/contentstack-bulk-publish/src/commands/cm/assets/publish.js

```diff
@@ -1,5 +1,5 @@
 const { Command } = require('@contentstack/cli-command');
```

Comment:
@vkalta Can you please update the version bump information and autogenerated readme file?

Reply:
@vkalta Is version bump required for this plugin?